### ISSUE TITLE
Click on the Grav blue icon in Admin panel (top of side menu) causes Error

### ISSUE THREAD

**Issue by mjmaax** [getgrav/grav#3262]
I get this error:

I use Grav 1.7.7 which I upgraded from 1.6.28 

ERROR: flex-objects.html.twig template not found for page:
Flex Objects

**Comment by mahagr** [comment]
I cannot reproduce.

Some old plugins/themes may replace template file lookup instead of adding their own path, so try to disable plugins one by one.

**Comment by mjmaax** [comment]
Sorry for the late reply. Was indeed a plugin which used to work well in 1.6.x but causes this error with 1.7.

The following line specifies the path where the templates should found for that plugin (to customize the tool section of Grav admin):

    public function onAdminTwigTemplatePaths($event)
    {
        $event['paths'] = [__DIR__ . '/admin/themes/grav/templates'];
    }

If I comment it out, I no longer have the "ERROR: flex-objects.html.twig template not found for page" error, but of course the customizations don't apply to the tools section..

How could this be solved?

**Comment by mahagr** [comment]
Yeah, the plugin doesn't work. In fact, it breaks even in Grav 1.6 depending on which other plugins you have installed.

The fix is:
```php
public function onAdminTwigTemplatePaths($event)
{
     $paths = $event['paths']
     $paths[] = [__DIR__ . '/admin/themes/grav/templates'];
     $event['paths'] = $paths;
}
```

Can you also create a bug report for the plugin that behaves badly?

**Comment by mjmaax** [comment]
Thank you! I wrote the plugin...

The issue is indeed described here:
https://learn.getgrav.org/17/advanced/grav-development/grav-17-upgrade-guide

I went through that section before upgrading, but it's very diffcult to guess that this was causing the problem. If the section shows the kind of error happening and then explains the cause, it should makes it easier for the users to find where to fix the code.

Ideally, it would be nice if Grav could detect earlier wrong use of the API.. I guess something could be done to secure its usage.
